onfido/live_photos.py

```diff
@@ -0,0 +1,20 @@
+"""Wrapper for the live photos endpoint."""
+import mimetypes
+
+from. api_resource import ApiResource
```

Comment:
Why is there a dot after `from`? Is this supposed to be a relative import? In this case move the dot to before `api_resource`, and add a space.